tests/test_environments/k8s-cloudbeat-tests/values.yaml

```diff
@@ -7,7 +7,7 @@ replicaCount: 1
 image:
   name: cloudbeat-test
   repository: cloudbeat-test
-  pullPolicy: Always
+  pullPolicy: IfNotPresent
```

Comment:
For k8s tests, the images are loaded to the nodes locally, so the pull policy should be `IfNotPresent`.